styles/globals.scss

```diff
@@ -57,6 +57,14 @@ body {
   text-align: center;
 }
 
+.app-global-alert {
+  background-color: #ffeb3b;
+}
+
+.app-global-alert__content {
+  padding-top: 24px;
```

Comment:
This will need tweaking if we add another type of alert, otherwise the padding between alerts is too much. It was added here instead of `.app-global-alert` to avoid a thin yellow strip being displayed when there was no `<CojAlert>` element.